applications/ParticleMechanicsApplication/python_scripts/particle_json_output_process.py

```diff
@@ -95,7 +95,7 @@ def ExecuteFinalizeSolutionStep(self):
                         values_vector = mp.CalculateOnIntegrationPoints(variable, self.sub_model_part.ProcessInfo)
                         value = values_vector[0]
 
-                        if (variable_type == "Double" or variable_type == "Component"):
+                        if (variable_type == "Double" or variable_type == "Integer" or variable_type == "Component"):
```

Comment:
```suggestion
                        if variable_type in ["Double", "Integer"]:
```

componets no longer exist